data/transactions/logic/doc.go

```diff
@@ -75,13 +75,15 @@ var opDocByName = map[string]string{
 	"intc_2":     "constant 2 from intcblock",
 	"intc_3":     "constant 3 from intcblock",
 	"pushint":    "immediate UINT",
+	"pushints":   "push sequence of immediate uints to stack",
```

Comment:
Should we add "(first int is pushed deepest)" or similar?

Reply:
going with: "push sequence of immediate uints to stack in the order they appear (first uint being deepest)"